test/kino/video_test.exs

```diff
@@ -4,15 +4,15 @@ defmodule Kino.VideoTest do
   describe "new/2" do
     test "raises an error for a non-image MIME type" do
       assert_raise ArgumentError,
-                   ~s{expected video type to be either :mp4, :ogg, :avi, :wmv, :mov, or an video MIME type string, got: "application/json"},
+                   ~s{expected video type to be either :mp4, :ogg, :avi, :wmv, :mov, or a video MIME type string, got: "application/json"},
```

Comment:
We need to adjust the raising code also :D

Reply:
True!